src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const minImg = document.querySelectorAll('a');
+const minImgArr = [...minImg];
+
+minImgArr.forEach((item, i) => {
+  minImg[i].addEventListener('click', (ev) => {
+    largeImg.src = item.getAttribute('href');
+    ev.preventDefault();
+  });
+});
```

Comment:
there should be only one event listener on container, on `.gallery` element
use `closest` selector to get href from link